.gitignore

```diff
@@ -14,4 +14,10 @@ coverage
 *.env
 .DS_Store
 packages/testkit/test-results
-packages/testkit/reports
\ No newline at end of file
+packages/testkit/reports
+
+# Bundle size analysis files
+base.json
+pr.json
+bundle-sizes.md
+.size-limit.js
```

Comment:
```suggestion
size-limit.js
```

?

Reply:
That's actually intended, that's the format (though it can be customized but it won't be relevant if I ignore the entire folder